modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -44,33 +46,33 @@
 
 	private static Log log = LogFactory.getLog(OperationMgtService.class);
 
-    @GET
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("{id}")
-    public List<? extends org.wso2.carbon.device.mgt.common.operation.mgt.Operation> getPendingOperations
-            (@HeaderParam("Accept") String acceptHeader, @PathParam("id") String id) {
-
-        Message message = new Message();
-        MediaType responseMediaType = AndroidAPIUtils.getResponseMediaType(acceptHeader);
-        java.util.List<? extends org.wso2.carbon.device.mgt.common.operation.mgt.Operation> operations = new
-                ArrayList<Operation>();
-        if (log.isDebugEnabled()) {
-            log.debug("Invoking Android pending operations:"+id);
-        }
-        DeviceIdentifier deviceIdentifier = AndroidAPIUtils.convertToDeviceIdentifierObject(id);
-        try {
-             operations = AndroidAPIUtils.getDeviceManagementService().getPendingOperations(deviceIdentifier);
-        }catch(OperationManagementException ex){
-            message.setResponseMessage("Issue in retrieving operation management service instance");
-            throw new AndroidOperationException(message, responseMediaType);
-        }
-        return operations;
-    }
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	@Path("{id}")
+	public List<? extends org.wso2.carbon.device.mgt.common.operation.mgt.Operation> getPendingOperations
+			(@HeaderParam("Accept") String acceptHeader, @PathParam("id") String id) {
+
+		Message message = new Message();
+		MediaType responseMediaType = AndroidAPIUtils.getResponseMediaType(acceptHeader);
+		java.util.List<? extends org.wso2.carbon.device.mgt.common.operation.mgt.Operation> operations = new
+				ArrayList<Operation>();
+		if (log.isDebugEnabled()) {
+			log.debug("Invoking Android pending operations:"+id);
+		}
+		DeviceIdentifier deviceIdentifier = AndroidAPIUtils.convertToDeviceIdentifierObject(id);
+		try {
+			operations = AndroidAPIUtils.getDeviceManagementService().getPendingOperations(deviceIdentifier);
+		}catch(OperationManagementException ex){
```

Comment:
Fixed in Commit [https://github.com/wso2/product-mdm/commit/a482e277cacdfa8e1795638d33f5275108d75069]